src/main/kotlin/frc/robot/subsystems/elevator/ElevatorIOReal.kt

```diff
@@ -0,0 +1,61 @@
+package frc.robot.subsystems.elevator
+
+import MOTOR_ID
+import com.ctre.phoenix6.configs.CurrentLimitsConfigs
+import com.ctre.phoenix6.configs.FeedbackConfigs
+import com.ctre.phoenix6.configs.MotorOutputConfigs
+import com.ctre.phoenix6.configs.Slot0Configs
+import com.ctre.phoenix6.configs.TalonFXConfiguration
+import com.ctre.phoenix6.controls.PositionVoltage
+import com.ctre.phoenix6.hardware.TalonFX
+import com.ctre.phoenix6.signals.InvertedValue
+import com.ctre.phoenix6.signals.NeutralModeValue
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.measure.Distance
+
+class ElevatorIOReal : ElevatorIO {
+    override val inputs = LoggedElevatorInputs()
+    private val motor = TalonFX(MOTOR_ID)
+    private val motorPosititonRequest = PositionVoltage(0.0)
+
+    init {
+        val motorConfig = TalonFXConfiguration().apply {
+            MotorOutput = MotorOutputConfigs().apply {
+                NeutralMode = NeutralModeValue.Brake
+                Inverted = InvertedValue.Clockwise_Positive
+            }
+            Feedback = FeedbackConfigs().apply {
+                RotorToSensorRatio = 1.0
+            }
+            Slot0 = Slot0Configs().apply {
+                kP = GAINS.kP
+                kI = GAINS.kI
+                kD = GAINS.kD
+            }
+            CurrentLimits = CurrentLimitsConfigs().apply {
+                StatorCurrentLimitEnable = true
+                SupplyCurrentLimitEnable = true
+                StatorCurrentLimit = 80.0
+                SupplyCurrentLimit = 40.0
+            }
+        }
+        motor.configurator.apply(motorConfig)
+    }
+
+    override fun setHeight(height: Distance) {
+        val rotationalPosition = Units.Rotations.of(height.`in`(Units.Centimeter) / ROTATIONS_TO_CENTIMETER)
+        motor.setControl(motorPosititonRequest.withPosition(rotationalPosition))
+    }
+
+    override fun setPower(percentOutput: Double) {
+        motor.set(percentOutput)
+    }
+
+    override fun reset() {
+        motor.setPosition(0.0)
+    }
+    override fun updateInputs() {
+        inputs.appliedVoltege = motor.motorVoltage.value
+        inputs.height = Units.Centimeter.of(motor.position.value.`in`(Units.Rotations) * ROTATIONS_TO_CENTIMETER)
```

Comment:
Can you find a way to make this shorter? Perhaps using [`Per`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/units/measure/Per.html). @rakrakon @emma03l your help will be appreciated here too, I didn't find a way to make it work. Basically we want a way to convert an angle measure (received from the motor) to a distance measure (height) without creating a new measure every loop.
At the very least, use a mutable measure. But I think with some `Per` magic and using `timesDivisor()` and/or `timesConversionFactor()` we can solve this. Thanks!